### PR TITLE
[DEV-2550] fix tests in social network

### PR DESCRIPTION
**Pull request by nonibansal**

### CODE REVIEW

File: fennel/client_tests/test_social_network.py [hunk 1]
```diff
@@ -151,8 +150,6 @@ def extractor_category_view(
         )
 
 
-@pytest.mark.skip(reason="Failing in main currently: TODO: DEV-2550")
-@pytest.mark.slow
```
Comment (aditya-nambiar) [inline fennel/client_tests/test_social_network.py:155]: Lets keep the @pytest.mark.slow marker, to make iteration speed faster while running tests locally

File: fennel/client_tests/test_social_network.py [hunk 2]
```diff
@@ -62,7 +61,7 @@ class CityInfo:
     @inputs(UserInfo)
     def count_city_gender(cls, user_info: Dataset):
         return user_info.groupby(["city", "gender"]).aggregate(
-            [Count(window=Window("1y 5s"), into_field="count")]
+            [Count(window=Window("6y 8s"), into_field="count")]
```
Comment (aditya-nambiar) [inline fennel/client_tests/test_social_network.py:64]: I believe we have removed the requirement to specify Window, so you can directly specify "6y 8s"

**Review by aditya-nambiar — APPROVED** [review]

File: fennel/client_tests/test_social_network.py (hunk 1 above)
Comment (nonibansal) [reply fennel/client_tests/test_social_network.py:155]: alright

File: fennel/client_tests/test_social_network.py (hunk 2 above)
Comment (nonibansal) [reply fennel/client_tests/test_social_network.py:64]: will change this.

**Pull request merged**